Source/WebCore/css/CSSProperties.json

```diff
@@ -1302,18 +1302,16 @@
         },
         "zoom": {
             "values": [
-                "normal",
-                "reset",
-                "document"
+                "normal"
             ],
             "codegen-properties": {
+                "initial": "initialZoom",
                 "custom": "All",
                 "high-priority": true,
-                "parser-grammar": "normal | reset | document | <percentage [0,inf]> | <number [0,inf]>"
+                "parser-grammar": "normal | <percentage [0,inf]> | <number [0,inf]>"
             },
-            "status": "non-standard",
             "specification": {
-                "url": "https://msdn.microsoft.com/en-us/library/ms531189(v=vs.85).aspx"
+                "url": "https://drafts.csswg.org/css-viewport/#zoom-property"
```

Comment:
Can you add `"category": "css-viewport"` here and define the category with the rest of the specs in this file?

Reply:
I tried adding it but it gave me style error that this category is unknown. Any reference PR, to see what else I need to do?

Reply:
This is what i meant by defining the category: https://searchfox.org/wubkat/rev/b36cbce69fddb7da33823f316bd8ead5bebee970/Source/WebCore/css/CSSProperties.json#9939

Reply:
Got it. Thanks!

Reply:
Done. :-) Thanks again for your guidance.